### PR TITLE
Support for Head Tracker sensor

### PR DESCRIPTION
**Pull request by Chandra-Mauli-Sharma**
In ref. to #38.

The support for the Head Tracker sensor is added.

### CODE REVIEW

**Comment by shubhamsinghmutualmobile** [comment]
@Chandra-Mauli-Sharma Could you please also add a screenshot/recording showcasing that the implementation is working fine on a device (be it an emulator or a physical device)?

File: composesensors/src/main/java/com/mutualmobile/composesensors/HeadTrackerSensorState.kt [hunk 1]
```diff
@@ -0,0 +1,104 @@
+package com.mutualmobile.composesensors
+
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.Immutable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+
+/**
+ * A sensor of this type measures the orientation of a user's head relative to an arbitrary reference frame, as well as the rate of rotation.
```
Comment (shubhamsinghmutualmobile) [inline composesensors/src/main/java/com/mutualmobile/composesensors/HeadTrackerSensorState.kt:10]: Replace `A sesor of this type` with `The HeadTracker sensor`

**Comment by Chandra-Mauli-Sharma** [comment]
> @Chandra-Mauli-Sharma Could you please also add a screenshot/recording showcasing that the implementation is working fine on a device (be it an emulator or a physical device)?

It was not working on my phone or emulator. I tried it on Pixel 7 but it was not supported in that too.

I also found this in the documentation: https://developer.android.com/reference/android/hardware/Sensor#TYPE_HEAD_TRACKER

image

**Review by shubhamsinghmutualmobile — APPROVED** [review]

**Comment by shubhamsinghmutualmobile** [comment]
Whenever we raise a PR that fixes an issue, it's good to use the `closing keywords` like (fixes #IssueNumber, or resolves #IssueNumber). It automatically connects our PR to that issue so that whenever our PR is merged, that issue is also closed ✅ ([Click here](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) for more info)

**Comment by Chandra-Mauli-Sharma** [comment]
Thanks for the info. It means a lot. next time I will take care.

**Pull request merged**